CHANGES_NEXT_RELEASE

```diff
@@ -12,3 +12,4 @@
 - Fix: POST /v2/subscriptions, attributes in notification are optional (#1952)
 - Add: Correlator as HTTP header and in log file (Issue #1916)
 - Fix: no longer considering service (tenant) for subscriptions when broker is started without '-multitenant' CLI option (Issue #1898)
+- Add: Add permanent subscriptions (Issue #1949)
```

Comment:
The following piece of documentation needs to be fixed (https://fiware-orion.readthedocs.org/en/develop/admin/database_model/index.html#csubs-collection):

> expiration: this is the timestamp on which the subscription expires. This is calculated using the duration parameter included in the subscribeContext operation (basically, sum "now" and duration) and will be recalculated when an updateContextSubscription is received (see programmers guide). For permanent subscriptions (allowed in NGSIv2) it is set to -1.

Maybe chaning 

For permanent subscriptions (allowed in NGSIv2) it is set to -1.

by

For permanent subscriptions (allowed in NGSIv2) an absurdely high value is used (see PERMANENT_SUBS_DATETIME in the source code)